lib/read.js

```diff
@@ -1,113 +1,87 @@
-
-module.exports = read
-
-var readline = require('readline')
-var Mute = require('mute-stream')
-
-function read (opts, cb) {
```

Comment:
promzard still uses the callback method and would need a similar refactor to use this new interface.

Reply:
promzard is getting a similar refactor here: https://github.com/npm/promzard/pull/35 but I have one more failing test to debug

Reply:
this will be a breaking change so i dont think we need to wait to release this. once im done with promzard i can update it to `read@2.0.0`